src/OpenTelemetry.Abstractions/Utils/Extensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <copyright file="Extensions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Utils
+{
+    using System;
+    using System.Globalization;
+    using System.Threading;
+
+    public static class Extensions
```

Comment:
please avoid making it public - it does not worth it. Copy/paste is better, compiling file into different projects is even better than that.

Reply:
What if we hide it from visual studio etc. with editorbrowsable=never?

Reply:
why? to avoid copy-paste or compiling the same file into two different projects?

still, users can use it and changes in the API are breaking, so why?